client/FileUpload.js

```diff
@@ -82,23 +102,29 @@ class FileUpload extends Component {
     render() {
         const classes = this.props.classes;
         const doc = this.state.fileDoc;
+        // tags are different in an image doc vs the bio photo in the contrib doc
+        const thumbnail = (this.state.isBioPhoto) ? 'bioThumbnail' : 'thumbnail';
+        const url = (this.state.isBioPhoto) ? 'bioUrl' : 'url';
+        const icon = (this.state.isBioPhoto) ? 'bioIcon' : 'icon';
+        const name = (this.state.isBioPhoto) ? 'bioName' : 'name';
+        const caption = 'caption'; // bio doesnt have a caption
         if(!doc) {
             return <div />;
         }
         const isVideo = (this.props.fileType === 'Video');
-        let fileUploadIcon = doc.url ?
-                ((doc.thumbnail || doc.icon) ?
-                    (<img className={classes.fabImg} alt="Upload preview" src={(doc.thumbnail || doc.icon)} />) :
+        let fileUploadIcon = doc[url] ?
```

Comment:
unused var

Reply:
fileUploadIcon seems to be used in line 135